### ISSUE TITLE
Chat message does not have a sender details ,

### ISSUE THREAD

**Issue by Goldilakhmani** [Tonterias/spingularchatsql6#23]
Chat message does not have a sender details ,it’s difficult to find the sender and who is receiver of message

**Comment by Tonterias** [comment]
Yes, you are absolutely right. Since the Photo entity was going to be used in the next version and separate it form JHI_User I was thinking we were going to add that photo over the Message, but we should not forget about this, so I will include it as a Feature Request for the next version v6.3.1?

Thanks
